infra/build/functions/build_lib.py

```diff
@@ -581,6 +581,6 @@ def run_build(  # pylint: disable=too-many-arguments
   build_id = build_info['metadata']['build']['id']
 
   logging.info('Build ID: %s', build_id)
-  logging.info('Logs: %s', get_logs_url(build_id, cloud_project))
+  logging.info('Logs: %s', get_logs_url(build_id))
   logging.info('Cloud build page: %s', get_gcb_url(build_id, cloud_project))
```

Comment:
Can we log just once per project, something like this:
logging.info(f'{project}. logs: {get_logs_url(build_id)}. GCB page: {get_gcb_url(build_id, cloud_project)}')